### PR TITLE
Fix .env problems, Display app name into type buttons, Use a proper foreground color instead of white for previews

### PR DESCRIPTION
**Pull request by heokhe**
# .env
On Linux, `USERNAME` env variable is taken:
```sh
echo $USERNAME
```
So I renamed `USERNAME` to `BOT_USERNAME` to stop the conflict. Also, you can use `API_ROOT` to specify a custom API root.
# Type buttons
When wanting to select the type of theme, The bot displays the proper app name:
image
# Fix text color in previews
Previously, The text in the preview image was not readable:
image
But now, The bot chooses a foreground color for each color (and white for dark colors):
image

### CODE REVIEW

**Comment by vrumger** [comment]
I think `USERNAME` can be removed if we change `bot.startPolling` to `bot.launch` as `launch` adds `botInfo` to `ctx`.

**Edit:** That might be a problem with [this line](https://github.com/YouTwitFace/ThemerBot/blob/ececb895acd140b6d11987a83de3b5482de7b818/variables/tgx-theme.js#L11).

File: index.js [hunk 1]
```diff
@@ -4,13 +4,19 @@ if (!process.env.TOKEN) {
...
 const {
     TOKEN,
-    USERNAME = ``,
+    BOT_USERNAME = ``,
     LOG_CHANNEL,
+    API_ROOT,
 } = process.env;
 
-const Telegraf = require(`telegraf`);
+const Telegraf = require(`telegraf/telegraf`);
```
Comment (vrumger) [inline index.js]: Why add `/telegraf`?

File: index.js [hunk 2]
```diff
@@ -4,13 +4,19 @@ if (!process.env.TOKEN) {
...
 const bot = new Telegraf(TOKEN, {
-    username: USERNAME,
+    username: BOT_USERNAME,
+    telegram: {
+        ...API_ROOT && {
+            apiRoot: `http://tgp.sh-sh.ir/`,
```
Comment (vrumger) [inline index.js]: I think you forgot to change this to `API_ROOT`.

Comment (heokhe) [reply index.js]: I didn't have enough focus on those days :joy: sorry for that

Anyway, I'm back at home and will fix these now.

**Comment by vrumger** [comment]
I think you can delete `variables/isLight.js` and import it from `variables/helpers.js`. It also looks like you haven't updated the themes with the primary color.

**Comment by vrumger** [comment]
I've contacted the translators for the new string. I'll send it here once I get them.

**Comment by vrumger** [comment]
[@ericksonx](https://t.me/ericksonx) says "I hope it has different bubble color for in and outcoming messages."

**Comment by heokhe** [comment]
I have a branch `primary` and it's many commits ahead of `master`. Updated Telegram X template with primary color is there. I'll start working on `.attheme` template today or tomorrow and when it is done, I'll make a new pull request.

---
Also, the idea for different bubble colors is a good idea! I'll change it now!

**Comment by vrumger** [comment]
@hkh12 Oh, great. 👍

**Comment by vrumger** [comment]
Any updates on this?

**Comment by heokhe** [comment]
I don't think so.
`primary` branch still exists, and TGX template is completed a long time ago. But attheme template is a bit harder to implement, and I've didn't even touch it :expressionless: btw I can send the pull request if you want it.

**Comment by vrumger** [comment]
Yeah, I think the `attheme` template is fine for now.

**Comment by heokhe** [comment]
So, may I send the pull request now?
What about this one? Will it get merged?

**Comment by vrumger** [comment]
Yes.

**Comment by heokhe** [comment]
Shouldn't this one get closed in favor of #9?

**Pull request closed without merging**